front/app/containers/Admin/pagesAndMenu/containers/GenericHeroBannerForm/SubmitWrapper/index.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { FormEvent, useRef, useEffect } from 'react';
```

Comment:
I copied the old version of the `SubmitWrapper` and put it here. Then, in the main one, I removed all the horrible `secondaryButton` stuff. It was only used in this single case so makes much more sense to duplicate.

Reply:
Agreed! Much cleaner :+1: